### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by ozgunozerk**

### CODE REVIEW

File: evm-template/node/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ edition = { workspace = true }
 license = { workspace = true }
 publish = false
 repository = { workspace = true }
-version = "0.1.2"
+version = "1.0.0"
```
Comment (ggonzalez94) [inline evm-template/node/Cargo.toml:10]: If we are doing a single release with the v2 tag does it make sense to keep separate versions for individual crates?

**Review by ggonzalez94 — APPROVED** [review]
LGTM! Just a small question regarding number version, which is non blocking and up to you

**Comment by ozgunozerk** [comment]
I say yes, the "single release" stuff may change. And starting "evm" template from version 2.0.0 seems like a bad practice.

These versions also communicate the history of them, so makese sense to generic one having a different version compared to evm imo.

**Review by KitHat — APPROVED** [review]

**Comment by KitHat** [comment]
We also need to bump the docs version to publish them on the docs.openzeppelin.com

**Pull request merged**